apps/dsp-app/src/app/project/image-settings/image-settings.component.html

```diff
@@ -0,0 +1,78 @@
+<dasch-swiss-centered-layout>
+  <dasch-swiss-app-progress-indicator *ngIf="isProjectsLoading$ | async"></dasch-swiss-app-progress-indicator>
+  <div class="content large middle" *ngIf="(isProjectsLoading$ | async) === false">
+    <div class="content-container">
+      <form *ngIf="form" class="form-content" [formGroup]="form">
+        <div class="large-field aspect">
+          <mat-slide-toggle formControlName="aspect">
+            {{ 'appLabels.form.project.imageSettings.aspect' | translate }}
+          </mat-slide-toggle>
+        </div>
+        <div *ngIf="form.controls.aspect.value" [@replaceAnimation]>
+          <div class="large-field">
+            <mat-label>
+              {{ 'appLabels.form.project.imageSettings.percentage-value' | translate }}: {{ percentageSlider.value}}%
+            </mat-label>
+          </div>
+          <mat-slider
+            min="1"
+            max="100"
+            step="1"
+            showTickMarks
+            discrete
+            [displayWith]="formatPercentageLabel"
+            ngDefaultControl
+            formControlName="percentage">
+            <input #percentageSlider matSliderThumb [value]="form.controls.percentage.value" />
+          </mat-slider>
+        </div>
+        <div *ngIf="!form.controls.aspect.value" [@replaceAnimation]>
+          <div class="large-field">
+            <mat-label>
+              {{ 'appLabels.form.project.imageSettings.absolute-width' | translate }}: {{
+              projectImageSettings.absoluteWidth }} px
+            </mat-label>
+          </div>
+          <mat-slider
+            [min]="0"
+            [max]="4"
+            [step]="1"
+            showTickMarks
+            [displayWith]="formatAbsoluteLabel"
+            discrete
+            ngDefaultControl
+            formControlName="absoluteWidthIndex">
+            <input
+              (valueChange)="absoluteSliderChange($event)"
+              [value]="absoluteWidthIndex(projectImageSettings.absoluteWidth)"
+              matSliderThumb />
+          </mat-slider>
+        </div>
+      </form>
+      <div style="display: flex; justify-content: space-between">
```

Comment:
so no class ?
Honestly we can keep the style, I think for now it's definitely better and easier for those flex properties